RELEASE_CADENCE.md

```diff
@@ -0,0 +1,107 @@
+# ZIO-HTTP(zHttp) Release Cadence
+
+- [Pre-Release cycles](#pre-release-cycles)
+  - [Release Highlights](#release-highlights)
+  - [Release Process](#release-process)
+  - [A Release Example](#a-release-example)
+
+- [Versioning](#versioning)
+- [Change log Templates](#change-log-templates)
+  - [Guiding Principle](#guiding-principle)
+  - [Example of a Changelog](#example-of-a-changelog)
+
+# Pre-Release cycles
+
+### Release Highlights
+* ZIO Http will follow a fortnightly release before first major release
+* We aim for regular and strict one release per 2 weeks.
+* 2 weeks are counted from the first release candidate to another.
+* This means that there is no code freeze and no feature should block a release
+
+### Release Process
+* We will target releases every second Wednesday. This will give us time to find some major bugs and do a patch release if required in the next 2-3 days
+* From release/next we will cut a branch following this pattern
```

Comment:
This line is not needed we will not have any release/next. All PR's will be opened against main